packages/docs/verite/patterns/smart-contract-verite.md

```diff
@@ -35,11 +37,11 @@ Upon completing successful verification of Verifiable Credentials, verifiers cre
 - The public on-chain address associated with the verified subject, and
 - The expiration time of the result (which may occur sooner than the expiration of the credential)
 
-A Verification Result created by a verifier is never persisted to a blockchain.
+A Verite `Verification Result` created by a verifier should never be persisted directly to a blockchain.
```

Comment:
I wonder, how does this work with persisting these info for audit trail ? Should those use some sort of IPFS instead ? 

Is the rationale behind this the privacy concern or are there other reasons ?

Reply:
The existing sentence is probably confusing. It's intended to be more of a statement of fact. 

If stored on-chain, the `Verification Result` is converted to a `Verification Record`. In some areas of the docs, we likely talk informally about how the payloads in the Result can have more data than what is ultimately persisted on-chain, and the idea was privacy concerns.

However, in practice, what matters is the signed part, which the smart contract needs to validate.

This use case is not immediately covered by the reference impl and may just cause confusion.

Maybe something like: 
"A Verification Result created by a verifier isn't directly persisted to a blockchain, but it may be (if on-chain persistence is needed by the use case) persisted as a minimized Verification Record"

Reply:
Thanks for the clarification, the updated statement does answer my initial question.

Reply:
good edit.  committed.